### PR TITLE
When only one item, label shouldn't be plural

### PR DESCRIPTION
**Pull request by bdukes**
Adds tests for logging the number of items at the end of the stream

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -38,7 +38,9 @@ module.exports = function (opts) {
 
 		cb(null, file);
 	}, function (cb) {
-		gutil.log(opts.title + ' ' + chalk.green(count + ' items'));
+		var plural = count != 1;
+		var itemsLabel = plural ? ' items' : ' item';
```
Comment (sindresorhus) [inline index.js]: Just use https://github.com/sindresorhus/plur

File: test.js [hunk 2]
```diff
@@ -44,3 +44,42 @@ it('should output debug info', function (cb) {
...
+
+it('should output singular item count', function (cb) {
+	var stream = debug({title: 'unicorn:'});
+
+	stream.on('finish', function () {
+		assert.strictEqual(stripAnsi(gutilStub.log.lastCall.args[0]).split('\n')[0],  'unicorn: 1 item');
+		cb();
+	});
+
+	stream.write(file, 'utf8', function () {
```
Comment (sindresorhus) [inline test.js]: Pretty sure it defaults to `utf8` when a string, so this is pretty moot.

**Comment by sindresorhus** [comment]
Tests are failing.

**Comment by bdukes** [comment]
The tests are failing on the use of [tildify](https://github.com/sindresorhus/tildify); I didn't change that part of the code

**Comment by bdukes** [comment]
Thanks!

**Pull request closed without merging**